ospd/command/command.py

```diff
@@ -253,9 +253,23 @@ def handle_xml(self, xml: Element) -> bytes:
 
         @return: Response string for <get_scanner_details> command.
         """
+        list_all = xml.get('list_all')
+        if list_all and list_all == '1':
+            list_all = True
+        else:
+            list_all = False
+
```

Comment:
This  could be replaced with
```suggestion
list_all = xml.get('list_all')
list_all = True if list_all == '1' else False
```